RoutesApp_ios/utilities/helpers/ConstantVariables.swift

```diff
@@ -100,5 +104,13 @@ class ConstantVariables {
 
     // Transformer Name
     static let transformerName = "CoordinateTransformer"
+     
+    // Google Maps keys
+    static let gmsServicesProvideAPIKeyProduction = ""
+    static let gmsPlacesClientProvideAPIKeyProduction  = ""
+    static let gmsServicesProvideAPIKeyDevelopment = ""
+    static let gmsPlacesClientProvideAPIKeyDevelopment = ""
```

Comment:
Is it possible to call the static variables from Env.swift?
To set the keys here

Reply:
they already have an Env.swift file on the project

Reply:
done:)